pkgs/development/python-modules/pygal/default.nix

```diff
@@ -26,14 +26,11 @@ buildPythonPackage rec {
   buildInputs = [
     flask
     pyquery
-
-    # Should be a check input, but upstream lists it under "setup_requires".
-    # https://github.com/Kozea/pygal/issues/430
-    pytestrunner
+    (pytestrunner.override {pytest = pytest_3;})
   ];
 
   checkInputs = [
-    pytest
+    pytest_3
```

Comment:
I think it's not needed to bump to pytest_3, because we could fix setup.cfg:
```diff
diff --git i/pkgs/development/python-modules/pygal/default.nix w/pkgs/development/python-modules/pygal/default.nix
index b0d533f5d68..e8b82d53d93 100644
--- i/pkgs/development/python-modules/pygal/default.nix
+++ w/pkgs/development/python-modules/pygal/default.nix
@@ -39,6 +39,8 @@ buildPythonPackage rec {
   preCheck = ''
     # necessary on darwin to pass the testsuite
     export LANG=en_US.UTF-8
+    # Fix setup.cfg for recent pytest versions. Remove with the next release
+    substituteInPlace setup.cfg --replace "[pytest]" "[tool:pytest]"
   '';
 
   propagatedBuildInputs = [ cairosvg tinycss cssselect ]
```
it was already [fixed in upstream](https://github.com/Kozea/pygal/commit/baeb3eb5540c1664d3c189b20cdf9e9595bbe012), but the patch doesn't apply cleanly. Besides that LGTM

Reply:
Hah - I did it via a substitute originally but someone told me "why not just use `pytest_3`?"...

Reply:
I'm inclined to use `substituteInPlace` because: 1. it's already fixed in upstream 2. we might forget the reasons behind using `pytest_3` and leave it in future version bumps.

Reply:
Okay, but it will make the other packages I fixed in this PR effectively unrelated - I fixed them to allow the `pytest_3` use to work.